packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -45,7 +45,8 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
 
     props.onNotificationClick?.(props.notification);
     if (props.notification.redirect?.url) {
-      window.open(props.notification.redirect?.url, '_blank', 'noreferrer noopener');
+      const target = props.notification.redirect?.target || '_blank';
+      window.open(props.notification.redirect?.url, target, 'noreferrer noopener');
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** please move the default target and default window features to a constants and please reuse them